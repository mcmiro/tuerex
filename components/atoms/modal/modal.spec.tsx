import * as React from "react";
import {screen, render} from "@testing-library/react";
import Modal from "./";

describe("Test Modal component", () => {
	it("Should pass when modal is defined", () => {
		render(<Modal data-testid="modal-test" />);
		const megaMenu = screen.queryByTestId("modal-test");
		expect(megaMenu).toBeDefined();
		expect(megaMenu).toMatchSnapshot();
	});
});
