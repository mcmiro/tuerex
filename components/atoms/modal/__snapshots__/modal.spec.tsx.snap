// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Modal component Should pass when modal is defined 1`] = `null`;
